### ISSUE TITLE
APP instead of BOTTLE

### ISSUE THREAD

**Issue by bikeridercz** [Gcenx/homebrew-wine#74]
Dear friend, is it possible to publish wine-crossover in old-fashioned way, ie. as an APP package instead of bottle ? Thanks.

**Comment by Gcenx** [comment]
This tap provides both a formula and cask, the README.md provides the command to install the cask.

```
brew install --cask --no-quarantine gcenx/wine/wine-crossover
```

**Comment by bikeridercz** [comment]
Yes I understand that, I prefere APP bacause I do not use a brew (I'm using Macports). Initially you provided app (Wine Crossover.app) so I'm just interested if there is that option in these days
